src/main/resources/theme/style-webview.css

```diff
@@ -130,6 +136,34 @@ img.country-flag[src=""] {
   color: #FFA500;
 }
 
+/* For compact chat mode */
+table {
+  width: 100%
+}
+
+td#text {
+  width: 99%;
+}
+
+td#author {
+  width: 18ch;
+  text-align: right;
+}
+
+td#avatar {
+  width: 44px;
+}
+
+td#flag {
+  width: 16px;
+}
+
+td#time {
+  width: 40px;
+  padding: 1em;
+  text-align: right;
+}
+
```

Comment:
I'm not sure I'm happy with assuming an 18ch width for the author name, 99% for text and 40px for time.
These things all need to scale well with different window sizes and zoom levels - do they?

I see two additional solutions:
1. We do it like it was initially, the whole chat is one table so whichever username is the longest will determine the width of the username column. All text will then be aligned the same:
```
Short Name       Hi all
Some long name   Hi!
```

2. Or, like DIscord does, texts are not aligned at all:

![image](https://user-images.githubusercontent.com/1864183/38454676-e5d0c3dc-3a6b-11e8-9ffb-db995fc64abe.png)

Which one of the 3 do we prefer? I think I'd go with the DIscord solution.

Reply:
I dont know, I dont see what a like or dislike about it. I might be the wrong person to ask about design aspects :D

Reply:
I think @20seven implementation is the most straight forward one, if I think about it. I mean it really supports readability and the amount of space wasted is ok I guess. Of curse only because it is this way now(username length), it won't be always like this, but in this very case I think it is neglectable as username length is not gonna ever change . And if so this is not the only playe where username length is limited(game team card thingy).

Reply:
I don'tuse the chat very often. Since a few community members stated, they don't switch to the java client because of the superior chat in python one, I'd go with it's design for opportunistic reasons

Reply:
I suggest to put the flag right after the user name, orherwise the space may be to wide to collate them easily. For badges this doesn‘t matter so much.

After all the chat can be styled with a theme, correct? So we could provide two different styles - one of them matching the Python client chat style.